Index: application/admin/controller/Common.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- application/admin/controller/Common.php	(revision )
+++ application/admin/controller/Common.php	(revision )
@@ -0,0 +1,20 @@
+<?php
+
+namespace app\admin\controller;
+
+use think\Controller;
+use think\Request;
+
+class Common extends Controller
+{
+    //
+    public function  __construct(Request $request=null)
+    {
+        parent::__construct($request);
+        //执行登陆验证
+        if(!session('admin.admin_id'))
+        {
+            $this->redirect('admin/login/login');
+        }
+    }
+}
